src/WowzaApiClient.php

```diff
@@ -40,4 +40,28 @@ public function setWowzaConfig($wowzaConfig)
     {
         $this->wowzaConfig = $wowzaConfig;
     }
+
+    /**
+     * @param Config $wowzaConfig
+     * @return Integer
+     */
+    public function checkWowzaConfig($wowzaConfig) {
```

Comment:
can you use type hints here?

Reply:
Am i not using type hints? Or where/what do you mean?